### ISSUE TITLE
Eslint does not charge and no apply validations

### ISSUE THREAD

**Issue by jsilveira2** [microsoft/vscode-eslint#856]
Hi;
I have a problem with using eslint in the visual code. I am currently trying to use it within a repository that contains more than one project. In total within this repository I have 4 package.json; One for the repository and three for each project that are basically front; api; and loopback;

When I do one of these three projects individually in my workspace eslint loads and applies validations normally. However if I put in my workspace the repository containing the three projects eslint does not seem to load and does not apply validations to any of the three projects.

I realize that it does not seem to load the library. Because when I run eslint Show outputs and it appears:

```
[Info - 11:26:07] ESLint server stopped.
[Info - 11:26:09] ESLint server running in node v12.4.0
[Info - 11:26:09] ESLint server is running.
```

And it never appears that eslint was loaded.

However if I do the procedure with one of the projects placed directly in my workspace always appears the line ESLint library loaded from: project / Path

**My configs**:
eslint version in all projects: ^6.7.2;
vscode-eslint extension: 2.0.10;
vscode: 1.41.1

To better illustrate.
**Not work:**
![image](https://user-images.githubusercontent.com/33058420/71259484-537ad400-2317-11ea-834f-6d3caf5cc439.png)

**Work:**
![image](https://user-images.githubusercontent.com/33058420/71259647-abb1d600-2317-11ea-9d90-726ad0ab673d.png)

if anyone can help me understand what is going on i appreciate it.
Sorry for my bad english.

**Comment by dbaeumer** [comment]
This got fixed in the latest update `2.0.10`. 

Dups https://github.com/microsoft/vscode-eslint/issues/854

Sorry for that. You could alternatively set the working direcotries like

```
"eslint,workingDirectories": [
  { "mode": "auto" }
]
```

**Comment by jsilveira2** [comment]
Thanks @dbaeumer 
I updated the version to 2.0.11 and solved my problem;
